tutorials/advanced_features/Backends for Sequence Execution.ipynb

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
<p>Would it also be possible to simply do: <code>emu_tn_config.sampling_rate = 0.5</code> without using the <code>replace</code> method?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
No, because it's a frozen dataclass...

Reply:
What is the rationale behind making it frozen, given that you need to change it anyway?

Reply:
The same behind not having `job_params = []` in the signature - so that we avoid having mutable objects as the default for keyword arguments. Since we are using `config=EmulatorConfig()` (or variations thereof) in the signature of the emulator backends `__init__`, I figured it would be safer to make the `EmulatorConfig` dataclass frozen.